ega-charts/localega/templates/doa-deploy.yaml

```diff
@@ -0,0 +1,112 @@
+{{- if .Values.doa.deploy }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "localega.fullname" . }}-doa
+  labels:
+    role: doa
+    app: {{ template "localega.name" . }}-doa
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: "{{ .Values.doa.name }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.doa.replicaCount }}
+  revisionHistoryLimit: {{ default "3" .Values.revisionHistory }}
+  selector:
+    matchLabels:
+      app: {{ template "localega.name" . }}-doa
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "localega.name" . }}-doa
+        role: doa
+        release: {{ .Release.Name }}
+    spec:
+    {{- if .Values.rbacEnabled}}
+      serviceAccountName: {{ .Release.Name }}
+    {{- end }}
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+      containers:
+      - name: doa
+        image: "{{ .Values.doa.repository }}:{{ .Values.doa.imageTag }}"
+        imagePullPolicy: {{ .Values.doa.imagePullPolicy | quote }}
+        command: ["java", "-jar", "localega-doa.jar"]
+        securityContext:
+          allowPrivilegeEscalation: false
+        env:
+        - name: SPRING_PROFILES_ACTIVE
+          value: no-oss,LocalEGA
+        - name: DB_INSTANCE
+          value: "{{- if .Values.filedatabase.deploy }}{{ template "localega.name" . }}-filedatabase:{{ .Values.filedatabase.servicePort }}{{- else }}{{ .Values.config.filedatabase_host }}:{{ .Values.config.filedatabase_port }}{{- end }}"
```

Comment:
This should probably not point to `filedatabase`  but rather our postgres server.

Reply:
Nope, this service talks directly to the DB.

Reply:
Yes, correct.